src/unicodeMake.js

```diff
@@ -0,0 +1,59 @@
+/* eslint no-console:0 no-undef:0 */
+
+// This is an internal Node tool, not part of the KaTeX distribution,
+// whose purpose is to generate unicodeSymbols.js in this directory.
+// In this way, only this tool, and not the distribution/browser,
+// needs String's normalize function.
```

Comment:
Can you include when this tool should be run?

Reply:
Added.